src/collections/_documentation/platforms/php/index.md

```diff
@@ -269,6 +269,12 @@ The number of attempts that should be made to send an event before erroring
 and dropping it from the queue.
 By default this option is set to `6`.
 
+
+##### max_request_body_size
+
+This option represents the size limit in bytes beyond which the body of the request is not captured. Default value is `medium`. Possible values are `none` = never send body, `small`, `medium`, `alawys` = always send the body, no matter how large it is.
```

Comment:
```suggestion
This option represents the size limit in bytes beyond which the body of the request is not captured. The default value is `medium`. Possible values are `none` = never send body, `small`, `medium`, `always` = always send the body, no matter how large it is.
```

Reply:
Isn't this part of the unified API?

Reply:
Yes, kinda, but as always we are only going to introduce this reactively to SDKs once ppl ask for it and not proactively push it out to all SDKs. 

For PHP it was there in the old SDK so we needed to bring it back.